app/src/main/java/com/example/wearit/ui/FavoritesScreen.kt

```diff
@@ -0,0 +1,328 @@
+            FavoritesContent(
+                outfits = outfits,
+                getItemById = getItemById,
+                getItemPhotoByPhotoFilename = getItemPhotoByPhotoFilename
+            )
+
+        },
+        bottomBar = {
+            BottomBar(
+                goToPickerScreen = goToPickerScreen,
+                goToWardrobe = goToWardrobe
+            )
+        }
+    )
+
+}
+
+@SuppressLint("SuspiciousIndentation")
+@Composable
+fun FavoritesContent(
+    outfits: List<Outfit>,
+    getItemById: (Int) -> Item?,
+    getItemPhotoByPhotoFilename: (itemId: String) -> Bitmap
+) {
+    Column(modifier = Modifier.fillMaxWidth()) {
+        Row(modifier = Modifier.padding(top=30.dp, bottom = 20.dp, start = 35.dp, end = 35.dp)) {
+            MasterButton(
+                type = ButtonType.WHITE,
+                onClick = { /*todo*/ },
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .weight(1f),
+                icon = R.drawable.editing,
+                text = "EDIT",
+            )
+        }
+        LazyColumn(
+            modifier = Modifier.fillMaxWidth(),
+            contentPadding = PaddingValues(horizontal = 35.dp, vertical = 100.dp),
+            verticalArrangement = Arrangement.spacedBy(50.dp),
+        ) {
+            items(outfits) { outfit ->
+                Box(
+                    contentAlignment = Alignment.Center,
+                    modifier = Modifier
+                        .padding(5.dp, 5.dp)
+                        .clip(shape = RoundedCornerShape(50.dp))
+                        .fillMaxWidth()
+                        .border(
+                            width = 5.dp,
+                            color = MaterialTheme.colors.primary.copy(alpha = LocalContentAlpha.current),
+                            shape = RoundedCornerShape(50.dp)
+                        )
+
+                ) {
+
+                    MakeHorizontalPager(
+                        outfit.itemsInOutfit,
+                        getItemById,
+                        getItemPhotoByPhotoFilename
+                    );
+
+                }
+            }
+        }
+
+    }
+
+
+}
+
+
+@OptIn(ExperimentalPagerApi::class)
+@Composable
+fun MakeHorizontalPager(
+    itemsInOutfit: List<Int>,
+    getItemById: (Int) -> Item?,
+    getItemPhotoByPhotoFilename: (itemId: String) -> Bitmap
+
+) {
+    val pagerState = rememberPagerState(initialPage = 0)
+    val coroutineScope = rememberCoroutineScope()
+
+    Column(
+        horizontalAlignment = Alignment.CenterHorizontally,
+        modifier = Modifier.fillMaxWidth()
+    ) {
+        HorizontalPager(
+            count = itemsInOutfit.size,
+            state = pagerState,
+            modifier = Modifier
+                .fillMaxWidth()
+        ) { currentPage ->
+            Row {
+                val finalItem = getItemById(itemsInOutfit[currentPage])
+                Image(
```

Comment:
Myślę, ze lepiej będzie opakować ten Image w if'a. Czyli:
if (finalItem != null) {
//tutaj wyświetl Image, ale już bez !!
}
Dzięki temu jeśli będzie jakiś błąd i nie znajdzie takiego itemu, to appka nam nie wybuchnie